src/test/components/StackChart.test.js

```diff
@@ -48,154 +53,162 @@ const GRAPH_WIDTH =
   GRAPH_BASE_WIDTH + TIMELINE_MARGIN_LEFT + TIMELINE_MARGIN_RIGHT;
 const GRAPH_HEIGHT = 300;
 
-describe('StackChart', function() {
+xdescribe('StackChart with samples', function() {
```

Comment:
Please re-flag me for review after this test is re-enabled and I can take a look at the tests again.